lib/js/components/Outline/OutlineItem/OutlineItem.consts.ts

```diff
@@ -0,0 +1,13 @@
+export const OUTLINE_ITEM_SIZES = {
+	S: 'small',
```

Comment:
in other places we use full name, and look at #1 in https://bethinkteam.slack.com/archives/C012R8RM3A4/p1683805516330539?thread_ts=1683798058.915649&cid=C012R8RM3A4
```suggestion
	SMALL: 'small',
```